pyron-engine/src/main/scala/com/cloudentity/pyron/rule/ApiGroupMatcher.scala

```diff
@@ -3,21 +3,15 @@ package com.cloudentity.pyron.rule
 import com.cloudentity.pyron.domain.flow.GroupMatchCriteria
 
 object ApiGroupMatcher {
-  def makeMatch(hostOpt: Option[String], path: String, criteria: GroupMatchCriteria): Boolean = {
-    val domainMatches =
-      criteria.domains match {
-        case Some(domains) =>
-          hostOpt match {
-            case Some(host) =>
-              val hostWithoutPort = host.takeWhile(_ != ':')
-              domains.find(_.regex.findFirstMatchIn(hostWithoutPort).isDefined).isDefined
-            case None =>
-              false
-          }
-        case None => true
-      }
-
-    domainMatches && path.startsWith(criteria.basePathResolved.value)
-  }
 
-}
+  def makeMatch(hostOpt: Option[String], path: String, criteria: GroupMatchCriteria): Boolean =
```

Comment:
Why is it changed?

Reply:
&& is lazy, and it makes sense to have the shorter/easier computation on the left, to possibly falsify quickly and never need to compute right side. The reordering of case checks ... doesn't change anything, and I will put it in the same order as it was. Not sure why I reordered it, will fix.